### PR TITLE
Skip PAF Approval process if no PAFReviewerRole assigned

### LINKED ISSUE #3839: If no PAF Reviewer Roles created skip project approval
This makes the system more adaptable to different use cases.

### PR DESCRIPTION
**Pull request by sandeepsajan0**
Fixes #3839 

##  Assumptions

An organization only chooses one option either go with PAFReviewerRoles or without that. Changing in between might cause the issue with `Internal Approval` status projects. We will fix this in another issue if needed.

## Test Steps

 - [ ] Remove all PAFReviewerRoles from wagtail admin
 - [ ] Select a project in draft state, and fill paf
 - [ ] There should be a `Continue to next status` button in Actions to take. Also check `To do` message in the sidebar.
 - [ ] After confirmation, the project should move to the contracting stage.
 - [ ] There shouldn't be any `Internal Approval` stage.

Don't forget to add those PAFReviewerRoles back because we have already chosen that option and to avoid the issue explained in assumption above.

### CODE REVIEW

**Review by frjo — APPROVED** [review]
Nice work!

**Comment by wes-otf** [comment]
Following the test steps this didn't seem to work on the test instance, I removed all approver roles from users but the project still went to internal review. Let me know if I misunderstood a step of this:
Screenshot 2024-05-21 at 17 06 31
Screenshot 2024-05-21 at 17 06 44
Screenshot 2024-05-21 at 17 07 09

**Comment by sandeepsajan0** [comment]
@wes-otf You don't have to remove 'approval roles from user' instead just remove PAFReviewerRoles from Project Settings.
It should be in `Project > Project Settings > PAFReviewerRoles`. 
image

File: hypha/apply/projects/templates/application_projects/project_admin_detail.html
Comment (wes-otf) [inline hypha/apply/projects/templates/application_projects/project_admin_detail.html]: What are your thoughts on the verbiage here being changed to something like `Please ensure the Project Form is completed and you are ready to proceed to the next stage. This action cannot be reverted.` for a bit more clarity & to use the new "Project Form" vs "Project Approval Form" language?

**Comment by wes-otf** [comment]
hahahaha thanks for that assist, I had a feeling I wasn't taking the right steps there. All looks good though! Left a minor comment about wording but perfect functionality wise.

File: hypha/apply/projects/templates/application_projects/project_admin_detail.html
Comment (frjo) [reply hypha/apply/projects/templates/application_projects/project_admin_detail.html]: Updated the text now.

**Pull request merged**
